traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/ds/DeliveryService.java

```diff
@@ -76,13 +77,19 @@
 	@JsonIgnore
 	private final String domain;
 	@JsonIgnore
+	private final String tld;
+	@JsonIgnore
+	private final Pattern wildcardPattern = Pattern.compile("^\\(\\.\\*\\\\\\.\\|\\^\\)|^\\.\\*\\\\\\.|\\\\\\.\\.\\*");
```

Comment:
Just looking at this I can't tell what the _actual_ regex is supposed to be with all these escaping backslashes. Can you include a comment that includes the unescaped version of the regex?

Reply:
Included a comment that explains an unescaped version of the regex in 067427f975